redesign/about.html

```diff
@@ -0,0 +1,183 @@
+            <img src="./img/common/dwyl-heart-only-logo.png" alt="dwyl logo" class="w1-plus"/>
+            dwyl
+          </a>
+        </li>
+        <li class="ph5 pt5 pb2 tl dn-ns"><a href="/" class="white link">Home</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/portfolio" class="white link">Portfolio</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/values" class="white link">Values</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/team" class="white link">Team</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/app" class="white link">App
+          <span class="dwyl-bg-gold pa1 br2 f7">BETA</span></a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/blog" class="dwyl-mint link">Blog</a></li>
+        <li class="ph5 di-ns pt2 pb3 tl pv0-ns"><a href="/#contact" class="white link">Contact</a></li>
+      </ul>
+    </nav>
+
+    <section class="dwyl-dark-gray pt6 pb3 pb3-ns ph6-ns center shadow-4 tc">
+      <img src="./img/common/dwyl-heart-only-logo.png" alt="dwyl logo" class="w4 tc"/>
+      <h1 class="f1">Driven by Our Values</h1>
+    </section>
+
+    <section class="dwyl-dark-gray pt3 pb3 pb5-ns ph6-ns center tl w-60-l w-90">
+      <h2 class="dwyl-mint bb dwyl-b--gray pl3">dwyl</h2>
+      <p class="pl3 b mv1 f6">NOUN</p>
+      <div>
+        <p class="pl3 b dib mv1">Pronounciation:</p>
+        <p class="dib pl3 mv1 times-new-roman">/dwɪɫ/</p>
+      </div>
+      <div>
+        <p class="pl3 b dib mv1">Origin:</p>
+        <p class="dib pl3 i mv0 times-new-roman">Do What You Love</p>
+      </div>
+      <div>
+        <p class="pl3 b dib mv1 dib">1.</p>
+        <p class="dib pl2 i mv0 times-new-roman w-80 w-90-ns v-top pt1">
+          Community of people on a mission to change the world using technology.
+        </p>
+      </div>
+
+    </section>
+
+    <section class="dwyl-bg-dark-gray white pt5 pb4 pb5-ns ph6-ns ph4 center">
+      <h2 class="f2 mv0">More Than Just Software</h2>
+      <p class="istok-web">We solve real problems with technology. We are different by design.</p>
+      <button class="bn dib link mb3 ph3 pv2 dwyl-bg-yellow dwyl-dark-gray br2 f5 shadow-4 pointer istok-web">Join Our Community</button>
+      <button class="bn dib link ml2 mb3 ph3 pv2 dwyl-bg-mint white br2 f5 shadow-4 pointer istok-web">Meet the Team</button>
+
+      <h2 class="f2 mt5 mb2">We believe in:</h2>
+      <div class="dib w-45-ns">
+        <h3 class="dwyl-yellow mt1">People 🤝</h3>
+        <p class="lh-copy">
+          Our people are our primary focus, not profit. We have a
+          <a href="#" class="dwyl-yellow">worldwide community of over 350 creative technologists.</a>
+          We believe in inclusivity and accessibility and want to enrich the
+          lives of everyone in our community. That’s why we made
+          <a href="https://www.focushub.co.uk/" class"dwyl-yellow">a home for our London-based team at Focus Hub.</a>
```

Comment:
This `dwyl-yellow` is being overriden in my browser: Chrome (standard, not Canary) v. 60.0.3112.113
<img width="481" alt="screen shot 2017-09-10 at 17 09 54" src="https://user-images.githubusercontent.com/4185328/30250711-1359d19c-964b-11e7-8b90-3ebc85cbba8d.png">

Probably because it's the only link in this section that points to an `http` link and is recognised by the browser as such. Reset possibly required for links universally at the top of the stylesheet (haven't checked to see if you have one!)

Reply:
Have opened a separate issue on line height: https://github.com/dwyl/dwyl-site/issues/315

Reply:
There was a missing "=" after class!